buildspec/languageServerTests.yml

```diff
@@ -0,0 +1,21 @@
+version: 0.2
```

Comment:
Other files use "lsp" in the name, consider naming this file with the same pattern (`lspTests.yml`).

Reply:
lsp does not correctly describe the system, which is a language server that happens to handle the lsp protocol.

Reply:
LSP is the common moniker. Is there a risk of confusion? Is there a reasonable chance that any non-LSP lang server will be implemented?